### PR TITLE
Go linter fixes

### PR DESCRIPTION
**Pull request by fiksn**
Go linter was not running in pipeline, fix that and all the issues it reports

### CODE REVIEW

File: channelchecker/settings.go [hunk 1]
```diff
@@ -7,18 +7,21 @@ import (
 	entities "github.com/bolt-observer/agent/entities"
 )
 
+// GlobalSettings reepresents the settings per pubkey
```
Comment (satoshisound) [inline channelchecker/settings.go:10]: represents

File: filter/filter.go [hunk 2]
```diff
@@ -14,13 +14,15 @@ import (
 	"github.com/golang/glog"
 )
 
+// FileFilter sturct
```
Comment (satoshisound) [inline filter/filter.go:17]: struct

**Review by satoshisound — APPROVED** [review]
Two small spelling mistakes in the comments. Otherwise, looks good.

File: channelchecker/settings.go (hunk 1 above)
Comment (fiksn) [reply channelchecker/settings.go]: 👍

File: filter/filter.go (hunk 2 above)
Comment (fiksn) [reply filter/filter.go]: 👍

**Pull request merged**